Backend/DynamicQR/DynamicQR.Domain/Exceptions/QrCodeNotFoundException.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace DynamicQR.Domain.Exceptions;
+
+public class QrCodeNotFoundException : Exception
+{
+    public QrCodeNotFoundException(string organisationId, string id, Exception? innerException) : base($"No QR Code was found for the organisation with id {organisationId} and with QR code id {id}.", innerException)
```

Comment:
@jensdenbraber  
Op zich prima om een specifieke QrCodeNotFoundException toe te voegen, maar voor de uitbreidbaarheid is dit iets minder chill. 

Mijn voorstel zou dan zijn om dit te doen:

`QrCodeNotFoundException : ItemNotFoundException`
`ItemNotFoundException : Exception`

Bij de endpoints kunnen we dan namelijk gemakkelijk middleware schrijven op de `ItemNotFoundException` en dat omvormen naar een 404 Status Code

Reply:
yes dat is een goed idee :)